components/groups/modals/memberManagementModal.tsx

```diff
@@ -171,185 +186,224 @@ export function MemberManagementModal({
 
   const submitFormRef = useRef<(() => void) | null>(null);
 
-  return (
-    <dialog id={modalId} className="modal ">
-      <div className="flex flex-col items-center w-full h-full">
-        <div className="modal-box dark:bg-[#1D192D] bg-[#FFFFFF] rounded-[24px] max-w-[39rem] p-6 dark:text-white text-black">
-          <form method="dialog">
-            <button className="btn btn-sm btn-circle btn-ghost absolute right-2 top-2">✕</button>
-          </form>
-          <div className="flex justify-between items-center my-8">
-            <h3 className="text-2xl font-semibold">Members</h3>
-            <button
-              type="button"
-              className="btn btn-gradient xs:max-3xl:w-[140px] xxs:w-auto h-[52px] text-white rounded-[12px] btn-sm"
-              onClick={handleAddMember}
-            >
-              New member
-            </button>
-          </div>
+  const modalContent = (
+    <dialog
+      id={modalId}
+      className={`modal ${showMemberManagementModal ? 'modal-open' : ''}`}
+      style={{
+        position: 'fixed',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0,
+        zIndex: 9999,
+        backgroundColor: 'transparent',
+        padding: 0,
+        margin: 0,
+        height: '100vh',
+        width: '100vw',
+        display: showMemberManagementModal ? 'flex' : 'none',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
```

Comment:
we'll reformat this so every modal uses the same modal base instead of consistently redefining the same code. From their we will use tailwind rather then defining styles. This method just worked early on while we were fixing lots of code.